CHANGELOG.md

```diff
@@ -5,6 +5,37 @@ All notable changes to Zebra are documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org).
 
+## [Zebra 1.0.0-rc.8](https://github.com/ZcashFoundation/zebra/releases/tag/v1.0.0-rc.8) - 2023-05-10
+
+This release adds end of support feature which will make `zebrad` binary to halt after the support time of 16 weeks has passed.
```

Comment:
This could be a surprising change for some users, so we might want to explain that it's the same as what `zcashd` does.

```suggestion
Starting in this release, Zebra has implemented an "end of support" halt. Just like `zcashd`, the `zebrad` binary will stop running 16 weeks after the last release date.
```